### ISSUE TITLE
RabbitMQ: "All stable feature flags must be enabled after completing an upgrade"

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/deploy#488]
This alert message appears after logging into the management plugin UI. It links to https://www.rabbitmq.com/docs/feature-flags

Not sure if any action is needed or if we need to add instructions for when upgrading RabbitMQ.

**Comment by jpmckinney** [comment]
In the log file:

```
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0> Deprecated features: `management_metrics_collection`: Feature `management_metrics_collection` is deprecated.
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0> By default, this feature can still be used for now.
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0> Its use will not be permitted by default in a future minor RabbitMQ version and the feature will be removed from a future major RabbitMQ version; actual versions to be determined.
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0> To continue using this feature when it is not permitted by default, set the following parameter in your configuration:
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0>     "deprecated_features.permit.management_metrics_collection = true"
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0> To test RabbitMQ as if the feature was removed, set this in your configuration:
2024-05-06 08:13:50.073184+00:00 [warning] <0.28370.0>     "deprecated_features.permit.management_metrics_collection = false"
```

**Comment by jpmckinney** [comment]
Aha, need to run `rabbitmqctl enable_feature_flag all`. I've added this to docs.

I've also configured `deprecated_features.permit.management_metrics_collection = true`. 

https://www.rabbitmq.com/docs/prometheus is recommended, but there seems to be no schedule to remove the deprecated management metrics.

Note that the above message appears even when `deprecated_features.permit.management_metrics_collection = true` is in the configuation file (setting it to false does indeed disable metrics).
